runtime/ops/os/mod.rs

```diff
@@ -57,7 +57,10 @@ deno_core::extension!(
   deno_os_worker,
   ops_fn = deno_ops,
   middleware = |op| match op.name {
-    "op_exit" | "op_set_exit_code" => op.disable(),
+    "op_exit" | "op_set_exit_code" => deno_core::OpDecl {
```

Comment:
Why this?

Reply:
Because these two ops are used in `ext/node` and they should be noops in the web worker context